nx.json

```diff
@@ -0,0 +1,54 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "lint",
+          "build",
+          "test",
+          "test:contracts",
+          "prettier",
+          "solhint",
+          "test:forge",
+          "test:hardhat",
+          "test:solidity",
+          "test:typescript",
+          "build:solidity",
+          "build:typescript"
+        ]
+      }
+    }
+  },
+  "targetDefaults": {
+    "dev": {
+      "dependsOn": ["^build"]
+    },
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": ["{projectRoot}/dist"]
+    },
```

Comment:
Build depends on dependencies builds and it outputs to `dist`.  This is currently not true for all packages and we need to override this config in those packages or make them output only to dist whichever is easiest on a per package basis.

Reply:
if we enumerated more outputs here and some package didn't put any files in those dirs, would that be okay?

e.g.
```
"outputs": ["{projectRoot}/dist", "{projectRoot}/out", "{projectRoot}/abi", "{projectRoot}/types"]
```

Reply:
Yup.   What we want is
- Very confident we overshot the outputs
- very confident we overshot the inputs too

Since safety is more important than overoptimizing cache hits

Reply:
The other thing we need to find is anything that is actually an output but is outputed to folders captured by inputs.   E.g. if a step generates a config file in the root that could break the cache

Reply:
`gas-report.json` is a root output in `store` and `world`